src/report/entry.cpp

```diff
@@ -25,21 +25,6 @@ namespace report {
     return entry;
 }
 
-Entry &Entry::operator=(Entry value) {
-    nlohmann::json::operator=(value);
-    return *this;
-}
-
-Entry &Entry::operator[](std::string key) {
-    // Note: out_of_range could not happen because operator[] returns
-    // a null json object if the accessed key is missing
-    try {
-        return static_cast<Entry &>(nlohmann::json::operator[](key));
-    } catch (std::domain_error &) {
-        throw JsonDomainError();
-    }
-}
-
```

Comment:
*TODO* for self: understand the implication of this hunk &mdash; I have forgotten!